src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationServiceImpl REGISTRATION_SERVICE
+            = new RegistrationServiceImpl();
+    private User defaultCorrectUser;
+
+    @BeforeEach
+    void setUp() {
+        defaultCorrectUser = new User();
+        defaultCorrectUser.setAge(20);
+        defaultCorrectUser.setLogin("correct login");
+        defaultCorrectUser.setPassword("correctPassword");
+    }
+
+    @Test
+    void register_NullUser_NotOk() {
+        User user = null;
+        assertThrows(InvalidUserException.class,
+                () -> REGISTRATION_SERVICE.register(user));

```

Comment:
![image](https://user-images.githubusercontent.com/12864128/233322267-3034c233-c57b-4e80-9fa9-881ebcff3d4f.png)

Reply:
I can't understand properly. If I replaced with a lower method, exception will not throw.

Reply:
This checklist bullet point pertains to test for registering already existing user